normandy/recipes/tests/api/v2/test_api.py

```diff
@@ -733,21 +733,27 @@ def test_it_can_enable_recipes(self, api_client):
             assert recipe.enabled
 
         def test_cannot_enable_unapproved_recipes(self, api_client):
-            recipe = RecipeFactory(enabled=False)
+            recipe = RecipeFactory()
+
+            res = api_client.post('/api/v2/recipe/%s/enable/' % recipe.id)
+            assert res.status_code == 409
+            assert res.data['error'] == 'Cannot enable a recipe that is not approved.'
+
+        def test_cannot_reenable_recipes(self, api_client):
```

Comment:
To me "reenable" sounds like "enable after disabling". Would a better name for this be `test_cannot_enable_enabled_recipe`?

Reply:
Makes sense.